CharlieBackend.Business/Helpers/PasswordHelper.cs

```diff
@@ -57,5 +58,47 @@ private static Int32 Next(Int32 minValue, Int32 maxValue)
                 }
             }
         }
+
+        public static string PasswordValidation(string password)
+        {
+            char[] symbols = new[] { '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', ',', '+', '=', '[', ']', '{', '}', ';', ':', '<', '>', '|', '.', '/', '?', ',', '-', '>', '<', '+', '=', '~' };
```

Comment:
Special symbols check can be described with a regular expression too. Something like this:
```cs
var hasSpecialSymbol = new Regex(@"[#?!@$%^&*-]+");
```